backend/postgres.js

```diff
@@ -4,7 +4,17 @@ export const sequelize = new Sequelize(
   process.env.DEV
     ? "postgres://localhost/dashboard?sslmode=disable"
     : process.env.POSTGRES_URL,
-  process.env.DEV ? {} : { dialect: "postgres", dialectOptions: { ssl: true } },
+  process.env.DEV
+    ? {}
+    : {
+        dialect: "postgres",
+        dialectOptions: {
+          ssl: {
+            require: true,
+            rejectUnauthorized: false,
```

Comment:
❓ What effect does adding `rejectedUnauthorized: false` have? Does it downgrade any security we have today? I couldn't figure out what it means.

Reply:
Answering my own question, it appears `rejectedUnauthorized: false` means an `sslmode` requiring verification of the server certificate won't be used. This should be fine given that the default is `prefer` which requires encryption but not verification.

Refs:
- https://github.com/brianc/node-postgres/blob/4b229275cfe41ca17b7d69bd39f91ada0068a5d0/packages/pg-connection-string/README.md
- https://www.postgresql.org/docs/9.1/libpq-ssl.html

Reply:
Also, one more learning I had, is that if the connection string contains the sslmode, or any other ssl parameters, the ssl object being specified here will be ignored. So if in any production environment sslmode=verify-ca is being used, it will take precedence and this change should not be a downgrade.

Refs:
- >If you plan to use a combination of a database connection string from the environment and SSL settings in the config object directly, then you must avoid including any of sslcert, sslkey, sslrootcert, or sslmode in the connection string. If any of these options are used then the ssl object is replaced and any additional options provided there will be lost.

  https://node-postgres.com/features/ssl#usage-with-connectionstring